package.json

```diff
@@ -34,6 +34,12 @@
     "test:all": "yarn run test:unit && yarn run test:acceptance",
     "test": "yarn run lint && yarn run test:all"
   },
+  "nyc": {
+    "exclude": [
+      "**/*-spec.js",
+      "test"
```

Comment:
I'm not sure why I needed to add this, but when I was running unit tests locally, the test/utils folder decided to pop up